<?xml version="1.0" encoding="UTF-8"?>
<class xmlns="http://xml.phpdox.net/src" full="Symfony\Component\Process\Exception\ProcessTimedOutException" namespace="Symfony\Component\Process\Exception" name="ProcessTimedOutException" abstract="false" final="false" start="21" end="69">
  <file path="//var/www/html/vendor/symfony/process/Exception" file="ProcessTimedOutException.php" realpath="/var/www/html/vendor/symfony/process/Exception/ProcessTimedOutException.php" size="1677" time="2022-04-08T05:07:18+00:00" unixtime="1649394438" sha1="7bed6e96f41a9c813a9f71419e703905ae3d4872" relative="ar/www/html/vendor/symfony/process/Exception/ProcessTimedOutException.php"/>
  <docblock>
    <description compact="Exception that is thrown when a process times out."/>
    <author value="Johannes M. Schmitt &lt;schmittjoh@gmail.com&gt;"/>
  </docblock>
  <extends full="Symfony\Component\Process\Exception\RuntimeException" namespace="Symfony\Component\Process\Exception" name="RuntimeException"/>
  <constant type="integer" name="TYPE_GENERAL" value="1"/>
  <constant type="integer" name="TYPE_IDLE" value="2"/>
  <member name="process" visibility="private" static="false" line="26"/>
  <member name="timeoutType" visibility="private" static="false" line="27"/>
  <constructor name="__construct" start="29" end="39" abstract="false" final="false" static="false" visibility="public">
    <parameter name="process" byreference="false" type="object">
      <type full="Symfony\Component\Process\Process" namespace="Symfony\Component\Process" name="Process"/>
    </parameter>
    <parameter name="timeoutType" byreference="false" type="int"/>
  </constructor>
  <method name="getProcess" start="41" end="44" abstract="false" final="false" static="false" visibility="public"/>
  <method name="isGeneralTimeout" start="46" end="49" abstract="false" final="false" static="false" visibility="public"/>
  <method name="isIdleTimeout" start="51" end="54" abstract="false" final="false" static="false" visibility="public"/>
  <method name="getExceededTimeout" start="56" end="68" abstract="false" final="false" static="false" visibility="public"/>
  <parent full="Symfony\Component\Process\Exception\RuntimeException" namespace="Symfony\Component\Process\Exception" name="RuntimeException">
    <extends full="RuntimeException" namespace="" name="RuntimeException"/>
    <implements full="Symfony\Component\Process\Exception\ExceptionInterface" namespace="Symfony\Component\Process\Exception" name="ExceptionInterface"/>
  </parent>
  <parent full="RuntimeException" namespace="" name="RuntimeException">
    <extends full="Exception" namespace="" name="Exception"/>
    <member name="message" static="false" visibility="protected">
    <default>null</default>
    <docblock>
      <description compact="The exception message"/>
      <var type="string"/>
    </docblock>
  </member>
    <member name="code" static="false" visibility="protected">
    <default>null</default>
    <docblock>
      <description compact="The exception code"/>
      <var type="int"/>
    </docblock>
  </member>
    <member name="file" static="false" visibility="protected">
    <default>null</default>
    <docblock>
      <description compact="The filename where the exception was created"/>
      <var type="string"/>
    </docblock>
  </member>
    <member name="line" static="false" visibility="protected">
    <default>null</default>
    <docblock>
      <description compact="The line where the exception was created"/>
      <var type="int"/>
    </docblock>
  </member>
    <method name="getMessage" abstract="false" static="false" visibility="public" final="true">
    <docblock>
      <description compact=""/>
      <return type="string"/>
    </docblock>
  </method>
    <method name="getPrevious" abstract="false" static="false" visibility="public" final="true">
    <docblock>
      <description compact=""/>
      <return type="Throwable"/>
    </docblock>
  </method>
    <method name="getCode" abstract="false" static="false" visibility="public" final="true">
    <docblock>
      <description compact=""/>
      <return type="mixed"/>
    </docblock>
  </method>
    <method name="getFile" abstract="false" static="false" visibility="public" final="true">
    <docblock>
      <description compact=""/>
      <return type="string"/>
    </docblock>
  </method>
    <method name="getLine" abstract="false" static="false" visibility="public" final="true">
    <docblock>
      <description compact=""/>
      <return type="int"/>
    </docblock>
  </method>
    <method name="getTrace" abstract="false" static="false" visibility="public" final="true">
    <docblock>
      <description compact=""/>
      <return type="array"/>
    </docblock>
  </method>
    <method name="getTraceAsString" abstract="false" static="false" visibility="public" final="true">
    <docblock>
      <description compact=""/>
      <return type="string"/>
    </docblock>
  </method>
    <method name="__toString" abstract="false" static="false" visibility="public" final="false">
    <docblock>
      <description compact=""/>
      <return type="string"/>
    </docblock>
  </method>
  </parent>
  <parent full="Exception" namespace="" name="Exception">
    <member name="message" static="false" visibility="protected">
    <default>null</default>
    <docblock>
      <description compact="The exception message"/>
      <var type="string"/>
    </docblock>
  </member>
    <member name="code" static="false" visibility="protected">
    <default>null</default>
    <docblock>
      <description compact="The exception code"/>
      <var type="int"/>
    </docblock>
  </member>
    <member name="file" static="false" visibility="protected">
    <default>null</default>
    <docblock>
      <description compact="The filename where the exception was created"/>
      <var type="string"/>
    </docblock>
  </member>
    <member name="line" static="false" visibility="protected">
    <default>null</default>
    <docblock>
      <description compact="The line where the exception was created"/>
      <var type="int"/>
    </docblock>
  </member>
    <constructor name="__construct" abstract="false" static="false" visibility="public" final="false">
    <docblock>
      <description compact="Construct the exception"/>
      <return type="void"/>
    </docblock>
    <parameter name="message" optional="true" byreference="false" type="string"/>
    <parameter name="code" optional="true" byreference="false" type="int"/>
    <parameter name="previous" optional="true" byreference="false" type="object" class="Throwable"/>
  </constructor>
    <method name="getMessage" abstract="false" static="false" visibility="public" final="true">
    <docblock>
      <description compact="Gets the Exception message"/>
      <return type="string"/>
    </docblock>
  </method>
    <method name="getPrevious" abstract="false" static="false" visibility="public" final="true">
    <docblock>
      <description compact="Returns previous Exception"/>
      <return type="Throwable"/>
    </docblock>
  </method>
    <method name="getCode" abstract="false" static="false" visibility="public" final="true">
    <docblock>
      <description compact="Gets the Exception code"/>
      <return type="mixed"/>
    </docblock>
  </method>
    <method name="getFile" abstract="false" static="false" visibility="public" final="true">
    <docblock>
      <description compact="Gets the file in which the exception was created"/>
      <return type="string"/>
    </docblock>
  </method>
    <method name="getLine" abstract="false" static="false" visibility="public" final="true">
    <docblock>
      <description compact="Gets the line in which the exception was created"/>
      <return type="int"/>
    </docblock>
  </method>
    <method name="getTrace" abstract="false" static="false" visibility="public" final="true">
    <docblock>
      <description compact="Gets the stack trace"/>
      <return type="array"/>
    </docblock>
  </method>
    <method name="getTraceAsString" abstract="false" static="false" visibility="public" final="true">
    <docblock>
      <description compact="Gets the stack trace as a string"/>
      <return type="string"/>
    </docblock>
  </method>
    <method name="__toString" abstract="false" static="false" visibility="public" final="false">
    <docblock>
      <description compact="String representation of the exception"/>
      <return type="string"/>
    </docblock>
  </method>
  </parent>
  <interface full="Symfony\Component\Process\Exception\ExceptionInterface" namespace="Symfony\Component\Process\Exception" name="ExceptionInterface">
    <extends full="Throwable" namespace="" name="Throwable"/>
  </interface>
</class>
